website/blog/2021-04-26-7.14.0.md

```diff
@@ -0,0 +1,144 @@
+share_text: "Babel 7.14.0 Released"
+---
+
+Babel 7.14.0 is out!
+
+This release enables class fields and private methods by default (they were promoted to Stage 4 during the last TC39 meeting!), and adds brand checks for private fields and static class blocks to `@babel/preset-env`'s `shippedProposals` option.
+
+We added support for [async do expressions](https://github.com/tc39/proposal-async-do-expressions) (using `@babel/plugin-proposal-async-do-expressions`), which extends the [do expression](https://github.com/tc39/proposal-do-expressions) proposal.
+
+Thanks to [Sosuke Suzuki](https://github.com/sosukesuzuki) and [Pig Fang](https://github.com/g-plane), Babel can now handle TypeScript 4.3 features. `@babel/parser`  also has a new option to correctly parse TypeScript declaration files.
+
+Finally, we introduced a new option to make it easier to produce dual modules by compiling ECMAScript imports to CommonJS that follow Node.js semantics.
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.14.0).
+
+<!-- truncate -->
+
+If you or your company want to support Babel and the evolution of JavaScript, but aren't sure how, you can donate to us on our [Open Collective](https://opencollective.com/babel) and, better yet, work with us on the implementation of [new ECMAScript proposals](https://github.com/babel/proposals) directly! As a volunteer-driven project, we rely on the community's support to fund our efforts in supporting the wide range of JavaScript users. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss more!
+
+## Highlights
+
+### New class features enabled by default
+
+The class fields and private methods proposals just reached Stage 4 and will be officially included in ECMAScript 2022! This was more of a formality since the semantics were already finalized and they've already been implemented in all the major browsers.
+
+You can read more details about this new syntax on MDN ([public fields](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Public_class_fields), [private fields and methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields)).
+
+```javascript
+class Check {
+  static className = "Check";
+  
+  #value = 3; // # means private!
+  
+  get #double() {
+    return this.#value * 2;
+  }
+}
+```
+
+You can remove `@babel/plugin-proposal-class-properties` and `@babel/plugin-proposal-private-methods`, since they are now enabled by default in `@babel/preset-env`.
+
+If you are using `@babel/preset-env`'s [`shippedProposals`](https://babeljs.io/docs/en/babel-preset-env#shippedproposals) option, it now also includes the `@babel/plugin-proposal-private-property-in-object` and `@babel/plugin-proposal-class-static-block` plugins: you can safely remove them from your configuration.
+
+> ⚠️ Webpack [doesn't support](https://github.com/webpack/webpack/issues/10216) this syntax yet.
```

Comment:
Now webpack supports class fields since 5.36.0 https://github.com/webpack/webpack/releases/tag/v5.36.0

Reply:
Added it!!